src/libData/AccountStore/AccountStoreSCEvm.cpp

```diff
@@ -34,20 +43,73 @@
 #include "libUtils/GasConv.h"
 #include "libUtils/SafeMath.h"
 #include "libUtils/TimeUtils.h"
+#include "libUtils/Tracing.h"
 #include "libUtils/TxnExtras.h"
 
-template <class MAP>
-void AccountStoreSC<MAP>::EvmCallRunner(const INVOKE_TYPE /*invoke_type*/,  //
+namespace {
+
+// A temporary class
+
+template <typename T>
+class ArtemTextMapCarrier
```

Comment:
Not usual to have someone's name as part of a class

Reply:
That code should have been deleted, nice catch, it was merely an example of how exporting for a distributed trace is handled.

Will gladly remove.